_pages/views.md

```diff
@@ -989,6 +989,38 @@ The dataGrid widget offers the following event handlers:
 * onRemovedHandler - what events to perform after a row is removed/deleted
 * onSelectedHandler - what events to perform after a row is selected
 
+#### dataGrid Example
+
+For a dataGrid, there must be a collection of the document you want to show 
+inside the document you want to show them from. 
+
+```xml
+
+<collection type="aggregation" name="example" persistent="false">
```

Comment:
Can you remove the `persistent="false"` from your example, it might confuse people. This is not required and would not be used in most circumstances.